playbook.md

```diff
@@ -899,6 +899,11 @@ We use Google Apps for Work to manage calendars, write and share documents. Ther
 
 When we write new things, we try to save them in a sensible folder within the existing structure.
 
+The dxw naming structure for documents is:
+
+Year-Month-Day    Doc type   Title
+E.g. 2017-01-01 Order Summary blog.gov.uk
```

Comment:
@vanessa-w What should this date correspond to? The date it was originally created? The date it was sent? (if external)